pkgs/tools/networking/dae/default.nix

```diff
@@ -37,6 +41,25 @@ buildGoModule rec {
   # network required
   doCheck = false;
 
+  installPhase = ''
+    runHook preInstall
+    install -Dm555 "$GOPATH"/bin/dae $out/bin/dae
+    install -Dm444 install/dae.service $out/lib/systemd/system/dae.service
```

Comment:
Perhaps both the installation of the service and the steps in `postFixup` can be moved to the `postInstall` phase to avoid modifying the `installPhase`. Apart from that, it looks good to me.